main.go

```diff
@@ -16,31 +18,50 @@ var yggdDispatchSocketAddr string
 var yggdHandler string
 
 func main() {
-	// Get initialization values from the environment.
 	var ok bool
+
+	configFile, ok := os.LookupEnv("CONFIG_FILE")
+	if ok {
+		config, err := toml.LoadFile(configFile)
+		if err != nil {
+			fmt.Errorf("cannot load config: %w", err)
+			return
+		}
+
+		for _, value := range config.GetArray("env").([]string) {
+			split := strings.Split(value, "=")
+			os.Setenv(split[0], split[1])
+		}
+
+		os.Setenv("YGG_FORWARDER_HANDLER", strings.TrimSuffix(filepath.Base(configFile), filepath.Ext(configFile)))
```

Comment:
After the rebase, this should be:
```suggestion
		os.Setenv("FORWARDER_HANDLER", strings.TrimSuffix(filepath.Base(configFile), filepath.Ext(configFile)))
```

Reply:
Done.